action.yml

```
@@ -22,11 +22,11 @@ inputs:
     description: "Default branch"
     required: false
   commit_title:
-    description: "Commit title"
+    description: "PR title"
```

[suggest] `commit_title` is used for a commit message title. Could you please rewrite it?

```suggestion
    description: "Commit and PR title"
```